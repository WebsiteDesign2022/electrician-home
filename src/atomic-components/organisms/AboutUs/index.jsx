import React from 'react';
import SecondaryTitle from '../../molecules/SecondaryTitle';
import SecondaryDescription from '../../molecules/SecondaryDescription';
import './index.scss';

const AboutUs = () => (
  <div className="col-xxl-8 px-4 py-5 about-us">
    <div className="row flex-lg-row align-items-left g-5 py-5">
      <div className="col-lg-8">

        <SecondaryTitle titlePartOne="About Us & Why We are" titlePartTwo="The Best For the Job" />

        <SecondaryDescription 
            text="Vestibulum ac diam sit amet quam vehicula elementum sed sit amet dui. Vivamus suscipit tortor eget felis porttitor volutpat. Donec rutrum congue leo eget malesuada. Nulla quis lorem ut libero malesuada feugiat. Quisque velit nisi, pretium ut lacinia in, elementum id enim. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Donec velit neque, auctor sit amet aliquam vel, ullamcorper sit amet ligula. " />

      </div>
      <div className="col-lg-12">
        <div className="d-grid gap-2 d-md-flex justify-content-end">

          <button type="button" className="btn btn-dark btn-lg px-4 box-shadow-bottom rounded-0">More about Us â†’</button>
        
        </div>
      </div>
    </div>
  </div>
);

export default AboutUs;
