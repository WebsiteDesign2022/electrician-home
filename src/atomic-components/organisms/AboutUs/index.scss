.box-shadow-bottom {
    padding: 10px;
    box-shadow: 10px 10px #E5A208;
}

.about-us {
    background-color: #FDB409;
}
