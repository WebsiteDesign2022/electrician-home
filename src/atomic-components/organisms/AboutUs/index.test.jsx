import TestRenderer from 'react-test-renderer';

import AboutUs from '.';

describe("AboutUs", () => {
    test("renders AboutUs correctly", () => {
        const aboutUs = TestRenderer.create(
            <AboutUs/>
        );

        expect(aboutUs.toJSON()).toMatchSnapshot();
    });
});
