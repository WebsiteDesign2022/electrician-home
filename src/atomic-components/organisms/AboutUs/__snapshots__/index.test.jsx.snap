// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutUs renders AboutUs correctly 1`] = `
<div
  className="col-xxl-8 px-4 py-5 about-us"
>
  <div
    className="row flex-lg-row align-items-left g-5 py-5"
  >
    <div
      className="col-lg-8"
    >
      <div>
        <div
          className="arrow-top-left-black"
        />
        <h1
          align="left"
          className="display-5 lh-1 mb-3 justify-content-end"
        >
          About Us & Why We are
        </h1>
        <h1
          align="left"
          className="display-5 fw-bold lh-1 mb-3 align-content-start justify-content-end"
        >
          The Best For the Job
        </h1>
      </div>
      <p
        align="left"
        className="text-black"
      >
        Vestibulum ac diam sit amet quam vehicula elementum sed sit amet dui. Vivamus suscipit tortor eget felis porttitor volutpat. Donec rutrum congue leo eget malesuada. Nulla quis lorem ut libero malesuada feugiat. Quisque velit nisi, pretium ut lacinia in, elementum id enim. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Donec velit neque, auctor sit amet aliquam vel, ullamcorper sit amet ligula. 
      </p>
    </div>
    <div
      className="col-lg-12"
    >
      <div
        className="d-grid gap-2 d-md-flex justify-content-end"
      >
        <button
          className="btn btn-dark btn-lg px-4 box-shadow-bottom rounded-0"
          type="button"
        >
          More about Us â†’
        </button>
      </div>
    </div>
  </div>
</div>
`;
